jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/assembly/AssemblyManagerTest.java

```diff
@@ -129,47 +106,8 @@ public void testCopyMultipleInvalidVerifyGivenDockerfile(@Injectable final KitLo
         new File(getClass().getResource("/docker/Dockerfile_assembly_verify_copy_valid.test").getPath()),
         buildConfig, new Properties(),
         logger);
-
-    // @formatter:off
-    new Verifications() {{
-      logger.warn(anyString, (Object []) any); times = 1;
-    }};
-    //@formatter:on
-  }
-
-  @Test
-  public void testCopyInvalidVerifyGivenDockerfile(@Injectable final KitLogger logger) throws IOException {
-    BuildConfiguration buildConfig = createBuildConfig();
-
-    AssemblyManager.verifyAssemblyReferencedInDockerfile(
-        new File(getClass().getResource("/docker/Dockerfile_assembly_verify_copy_invalid.test").getPath()),
-        buildConfig, new Properties(),
-        logger);
-
-    // @formatter:off
-    new Verifications() {{
-      logger.warn(anyString, (Object []) any); times = 1;
-    }};
-    //@formatter:on
-  }
-
-  @Test
-  public void testCopyChownValidVerifyGivenDockerfile(@Injectable final KitLogger logger) throws IOException {
```

Comment:
Why is this test deleted?

Reply:
we have moved these three tests to parameterized test cases ,created a separate class.